plugins/DigitalLibrary/src/DigitalFactoryController.py

```diff
@@ -385,6 +385,19 @@ def _onEngineCreated() -> None:
     def _applicationInitializationFinished(self) -> None:
         self._supported_file_types = self._application.getInstance().getMeshFileHandler().getSupportedFileTypesRead()
 
+        # Although cura supports these, it's super confusing in this context to show them.
+        if "jpg" in self._supported_file_types:
+            del self._supported_file_types["jpg"]
+        if "jpeg" in self._supported_file_types:
+            del self._supported_file_types["jpeg"]
+        if "png" in self._supported_file_types:
+            del self._supported_file_types["png"]
+        if "bmp" in self._supported_file_types:
+            del self._supported_file_types["bmp"]
+        if "gif" in self._supported_file_types:
+            del self._supported_file_types["gif"]
```

Comment:
I'll accept either way because it's a matter of opinion, but perhaps this would've been nicer:

```
for extension in ("jpg", "jpeg", "png", "bmp", "gif"):
    if extension in self._supported_file_types:
        del self._supported_file_types[extension]
```